source/documentation/doc-Release_Notes/topics/ref-ovirt-4.5.2.adoc

```diff
@@ -0,0 +1,102 @@
+
+BZ#link:https://bugzilla.redhat.com/2101481[2101481]::
+The handling of core dumps during upgrade from previous Red Hat Virtualization versions to RHV 4.4 SP1 batch 1, has been fixed.
+
+BZ#link:https://bugzilla.redhat.com/2104115[2104115]::
+Previously, when importing a virtual machine with manual CPU pinning (pinned to a dedicated host), the manual pinning string was cleared, but the CPU pinning policy was not set to NONE. As a result, importing failed.
+In this release, the CPU pinning policy is set to NONE if the CPU pinning string is cleared, and importing succeeds.
+
+BZ#link:https://bugzilla.redhat.com/2105781[2105781]::
+The hosted-engine-ha binaries have been moved from /usr/share to /usr/libexec. As a result, the hosted-engine --clean-metadata command fails. With this release, 
+you must use the new path for the command to succeed: /usr/libexec/ovirt-hosted-engine-ha/ovirt-ha-agent
+
+BZ#link:https://bugzilla.redhat.com/2109923[2109923]::
+Previously, it was not possible to import templates from the Administration Portal.
+With this release, importing templates from the  Administration Portal is now possible.
+
+==== Enhancements
+
+This release of Red Hat Virtualization features the following enhancements:
+
+BZ#link:https://bugzilla.redhat.com/1793207[1793207]::
+A new warning has been added to the vdsm-tool to protect users from using the unsupported user_friendly_names multipath configuration. The following is an example of the output:
+
+$ vdsm-tool is-configured --module multipath
+WARNING: Invalid configuration: 'user_friendly_names' is
+enabled in multipath configuration:
+  section1 {
+    key1 value1
+    user_friendly_names yes
+    key2 value2
+  }
+  section2 {
+    user_friendly_names yes
+  }
+This configuration is not supported and may lead to storage domain corruption.
+
+BZ#link:https://bugzilla.redhat.com/2097536[2097536]::
+In this release, the rhv-log-collector-analyzer now provides a detailed output for each problematic image, including disk names, associated virtual machine, the host running the virtual machine, snapshots, and the current SPM. This makes it easier to identify problematic virtual machines and collect SOS reports for related systems.
+The detailed view is now the default, and the compact option can be set by using the –compact switch in the command line.
+
+BZ#link:https://bugzilla.redhat.com/2097560[2097560]::
+Expiration of ovirt-provider-ovn certificate is now checked regularly along with other RHV certificates (such as engine CA, engine, or hypervisors) and if ovirt-provider-ovn is going to expire or already expired, the warning or alert is raised to the audit log. To renew the ovirt-provider-ovn certificate, administators need to run engine-setup.
```

Comment:
Typo: administators > administrators. But should probably be "you" because it's like saying "users" or "customers". We assume that the users are admins.

I recommend removing the parentheses around "such as ..." and breaking the sentence after "hypervisors".